test/plugin/base_test.rb

```diff
@@ -40,6 +40,24 @@ def setup
   # generic attributes
   HOSTNAME = Socket.gethostname
 
+  WRITE_LOG_ENTRIES_URI = 'https://logging.googleapis.com/v2beta1/entries:write'
+
+  APPENGINE_SERVICE_NAME = 'appengine.googleapis.com'
```

Comment:
Just realized: why are we defining these constants here and not reusing the ones from `out_google_cloud.rb`? And if we do need to redefine them, why the naming inconsistency (`APPENGINE_SERVICE_NAME` here vs. `APPENGINE_SERVICE` in `out_google_cloud.rb`)?